docs/modules/commons/pages/expressions.adoc

```diff
@@ -294,6 +294,48 @@ Compress the input string to GZip and encode compressed bytes to Base64 format
 |`H4sIAAAAAAAAACvLLMtMKS0GANIHCdkHAAAA`
 |===
 
+== Hash calculations
+
+=== `calculateHash`
+
+Calculates the hash using the specified hashing algorithm
+
+[source, subs="+quotes"]
+----
+#{calculateHash($algorithm, $input)}
+----
+
+* *`$algorithm`* - hash algorithm (can use https://datatracker.ietf.org/doc/rfc1319[MD2], https://datatracker.ietf.org/doc/rfc1321[MD5], https://datatracker.ietf.org/doc/rfc3174[SHA-1], https://datatracker.ietf.org/doc/rfc6234[SHA-256], https://datatracker.ietf.org/doc/rfc6234[SHA-384], https://datatracker.ietf.org/doc/rfc6234[SHA-512])
+* *`$input`* - any string to calculate hash
+
+.Example of hash calculating for string using MD5
+|===
+|Expression |Result
+
+|`#{calculateHash(MD5, vividus)}`
+|`0a05ba6064ae7e5d6ee9818f85b666ad`
+|===
+
+=== `calculateFileHash`
+
+Calculates the file hash using the specified hashing algorithm
+
+[source, subs="+quotes"]
+----
+#{calculateHash($algorithm, $filePath)}
+----
+
+* *`$algorithm`* - hash algorithm (can use https://datatracker.ietf.org/doc/rfc1319[MD2], https://datatracker.ietf.org/doc/rfc1321[MD5], https://datatracker.ietf.org/doc/rfc3174[SHA-1], https://datatracker.ietf.org/doc/rfc6234[SHA-256], https://datatracker.ietf.org/doc/rfc6234[SHA-384], https://datatracker.ietf.org/doc/rfc6234[SHA-512])
+* *`$filePath`* - the fully qualified file name including parent folders and extension (e.g. `temp/some_file.txt`)
```

Comment:
it should be like here:
https://github.com/vividus-framework/vividus/blame/d866507c8e6f4b0ae1034e1271ad8547b442a6f9/docs/modules/plugins/pages/plugin-avro.adoc#L25